metadata/nz_building_outlines.xml

```diff
@@ -291,6 +221,61 @@ If you adapt this work in any way or include it in a collection, and publish, di
           </gmd:geographicElement>
         </gmd:EX_Extent>
       </gmd:extent>
+      <gmd:extent>
```

Comment:
Yes, it's required since it's the beginning extent tag for the next section. The full history metadata has it as well. I can't see any other issue here - maybe git is misinterpreting the comparison of the before and after code. Maybe it's because I changed the indenting? I'm not really sure.